pkg/stacks/relationship.go

```diff
@@ -48,16 +61,54 @@ type KindlyIdentifier interface {
 	GroupVersionKind() schema.GroupVersionKind
 }
 
+// MultiParentLabelPrefix returns the NS specific prefix of a multi-parent label
+// for resources co-owned by a set of Stacks.
```

Comment:
Originally when I read this, I interpreted it as saying that it would parse a label and return its namespace prefix. Maybe an example of an input and output would be helpful in clearly explaining the reason this method exists.